doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
```

Comment:
We're going to have a lot of fun documenting things with ES6 template strings :)